src/utils/getCrsRd.ts

```diff
@@ -0,0 +1,46 @@
+import proj4, { InterfaceCoordinates } from 'proj4'
+import L, { LatLng } from 'leaflet'
+
+const def = `+proj=sterea +lat_0=52.15616055555555 +lon_0=5.38763888888889 +k=0.9999079 +x_0=155000 +y_0=463000 +ellps=bessel +towgs84=565.4171,50.3319,465.5524,-0.398957,0.343988,-1.87740,4.0725 +units=m +no_defs'`
+const proj4RD = proj4('WGS84', def)
+
+const getCrsRd = (maxZoom = 16, zeroScale = 3440.64, scales: number[] = []) => {
```

Comment:
Both the name and the implementation of this function is very cryptic, I would add some comments to describe what it does and what the parameters change.